pom.xml

```diff
@@ -29,7 +29,7 @@
             <groupId>javax.servlet</groupId>
             <artifactId>servlet-api</artifactId>
             <version>2.5</version>
-            <scope>provided</scope>
+<!--            <scope>provided</scope>-->
```

Comment:
Are you use main?

Reply:
This way I can use both.  Main and Tomcat.